### PR TITLE
Remove some of codacy problems

### PR DESCRIPTION
**Pull request by Beacze**
About 100 checkstyle's warnings less.

### CODE REVIEW

File: Kitodo/src/main/java/de/sub/goobi/forms/GranularityForm.java [hunk 1]
```diff
@@ -178,6 +178,8 @@ public void downloadClick() {
                 case QUARTERS:
                     result.add(new SelectItem("years", Helper.getTranslation("granularity.years")));
                     break;
+                default:
```
Comment (stweil) [inline Kitodo/src/main/java/de/sub/goobi/forms/GranularityForm.java]: We don't expect this case, do we? So adding an assertion might be good here.

File: Kitodo/src/main/java/de/sub/goobi/forms/ProzesskopieForm.java [hunk 2]
```diff
@@ -1948,6 +1948,8 @@ public static String createAtstsl(String title, String author) {
                     case 4:
                         result.append(word.length() > 1 ? word.substring(0, 1) : word);
                         break;
+                    default:
```
Comment (stweil) [inline Kitodo/src/main/java/de/sub/goobi/forms/ProzesskopieForm.java]: See comment above.

File: Kitodo/src/main/java/de/sub/goobi/helper/WebDav.java [hunk 3]
```diff
@@ -279,9 +279,9 @@ private void saveTiffHeader(Process inProcess) {
     public int getAnzahlBaende(String inVerzeichnis) {
         try {
             User aktuellerBenutzer = Helper.getCurrentUser();
-            String VerzeichnisAlle = serviceManager.getUserService().getHomeDirectory(aktuellerBenutzer)
+            String verzeichnisAlle = serviceManager.getUserService().getHomeDirectory(aktuellerBenutzer)
```
Comment (stweil) [inline Kitodo/src/main/java/de/sub/goobi/helper/WebDav.java]: Use English name?

File: Kitodo/src/main/java/de/sub/goobi/importer/ImportZentralblatt.java [hunk 4]
```diff
@@ -49,7 +49,7 @@
  */
 public class ImportZentralblatt {
     private static final Logger myLogger = Logger.getLogger(ImportZentralblatt.class);
-    String Trennzeichen;
+    String trennzeichen;
```
Comment (stweil) [inline Kitodo/src/main/java/de/sub/goobi/importer/ImportZentralblatt.java]: English?

File: Kitodo/src/main/java/de/sub/goobi/metadaten/Metadaten.java [hunk 5]
```diff
@@ -1977,10 +1977,10 @@ public String TransliterierenPerson() {
                 Person mdDin = new Person(mdtDin);
                 Person mdIso = new Person(mdtIso);
 
-                mdDin.setFirstname(trans.transliterate_din(md.getFirstname()));
-                mdDin.setLastname(trans.transliterate_din(md.getLastname()));
-                mdIso.setFirstname(trans.transliterate_iso(md.getFirstname()));
-                mdIso.setLastname(trans.transliterate_iso(md.getLastname()));
+                mdDin.setFirstname(trans.transliterateDin(md.getFirstname()));
```
Comment (stweil) [inline Kitodo/src/main/java/de/sub/goobi/metadaten/Metadaten.java]: Maybe `transliterateDIN`, `transliterateISO` would be better here.

File: Kitodo/src/main/java/de/sub/goobi/helper/WebDav.java (hunk 3 above)
Comment (Beacze) [inline Kitodo/src/main/java/de/sub/goobi/helper/WebDav.java]: fixed

File: Kitodo/src/main/java/de/sub/goobi/importer/ImportZentralblatt.java (hunk 4 above)
Comment (Beacze) [inline Kitodo/src/main/java/de/sub/goobi/importer/ImportZentralblatt.java]: fixed

File: Kitodo/src/main/java/de/sub/goobi/metadaten/Metadaten.java (hunk 5 above)
Comment (Beacze) [inline Kitodo/src/main/java/de/sub/goobi/metadaten/Metadaten.java]: fixed

File: Kitodo/src/main/java/de/sub/goobi/forms/GranularityForm.java (hunk 1 above)
Comment (Beacze) [inline Kitodo/src/main/java/de/sub/goobi/forms/GranularityForm.java]: good point! done

**Pull request merged**